FlexMeasures/flexmeasures
# fix: set date range for calendar to be persistent on session and UI

## Description

Fixed bug with date range not persisting across pages.

## Look & Feel
![Screenshot from 2024-09-06 10-24-41](https://github.com/user-attachments/assets/693ff035-5654-4af9-8f07-ef705780fbef)
![Screenshot from 2024-09-06 10-29-33](https://github.com/user-attachments/assets/accea337-14aa-425f-b507-fba3a0265e6a)

## How to test

- Navigate to your accounts page.
- Select an asset.
- Choose a date range.
- Open one sensor.
- Verify if the date range has been persisted.

## Further Improvements
None

## Related Items
None

---

- [✅] I agree to contribute to the project under Apache 2 License. 
- [✅] To the best of my knowledge, the proposed patch is not based on code under GPL or other license that is incompatible with FlexMeasures
